packages/frontapp/src/app/app.tsx

```diff
@@ -78,7 +78,7 @@ const App: FC = () => {
 
   const handleOnClickButton = useCallback(() => {
     setIsShowScanner(true);
-  }, []);
+  }, [setIsShowScanner]);
```

Comment:
I checked, that this is the most probably an error in eslint or somewhere else. We should not use hook setter as dependencies.